ocaml/typing/solver_intf.mli

```diff
@@ -296,13 +298,25 @@ module type Solver_polarized = sig
 
   (** Generalizes all reachable variables whose level is above [current_level],
       whose value is fully determined, by putting their level to [generic_level].*)
-      val generalize_structure :
-      current_level:int ->
-      generic_level:int ->
-      'a obj ->
-      ('a, 'l * 'r) mode ->
-      log:changes ref option ->
-      unit
+  val generalize_structure :
+    current_level:int ->
+    generic_level:int ->
+    'a obj ->
+    ('a, 'l * 'r) mode ->
+    log:changes ref option ->
+    unit
+
+  (** Copies all reachable variables whose level is at or above [copy_from_level],
+      and optionally puts the level of the copy to [copy_to_level].
```

Comment:
I think "puts" here is ambiguous, it would be to explicitly say that the level is forced to be less than or equal to [copy_to_level] to make clear that if something already has a level lower than that then it is left alone.